index.bs

```diff
@@ -263,6 +263,13 @@ so that it works across screen sizes,
 and allows users to choose their own font and font size without causing text to overflow.
 </div>
 
+Sometimes features aren't yet implemented on some platforms
+despite working on others.
+Design these cases so that
+code written to work on browsers where the feature isn't yet implemented
+also works on the missing platforms.
+See [[#feature-detect]].
+
```

Comment:
Might be useful to have an example here.

Reply:
Created an issue: https://github.com/w3ctag/design-principles/issues/428